src/components/color-picker/ColorPicker.tsx

```diff
@@ -0,0 +1,47 @@
+'use client';
+import React from 'react';
+import { CirclePicker, ColorResult } from 'react-color';
+import { Flex } from '@radix-ui/themes';
+const pickerColors = [
```

Comment:
Lets try meiteining consistent code style, please?
Could you put the constant to ceparate file `constants.ts` and type to `types.ts`.